docs/superpowers/plans/2026-05-07-sonarcloud-duplicated-lines.md

```
@@ -0,0 +1,369 @@
+Expected: `MISSING`
+
+If it exists already, STOP and report — the spec assumes clean creation.
+
+- [ ] **Step 2: Verify the only directories named `tests` are the intentional ones**
+
+This guards the `**/tests/**` glob from accidentally excluding production code.
+
+Run: `find . -type d -name tests -not -path '*/.venv/*' -not -path '*/node_modules/*' -not -path '*/.git/*'`
+Expected output (exact, in any order):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)